src/data/gods/hera.ts

```diff
@@ -136,11 +187,137 @@ const uncommonGrace: Boon = {
 const dyingWish: Boon = {
   name: "Dying Wish",
   type: OTHER,
+  element: EARTH,
+  info: (value) =>
+    `Whenever [hitch]-afflicted foes are slain, damage all other [hitch] afflicted foes for ${value}`,
+  values: {
+    [COMMON]: "40",
+  },
+  prerequisites: [[attack, special, dash, nastyComeback]],
+};
+
+export const braveFace: Boon = {
+  name: "Brave Face",
+  type: OTHER,
+  element: EARTH,
+  info: (value) =>
+    `Automatically use ${value} magic to resist up to 50% of any damage`,
+  values: {
+    [LEGENDARY]: { 1: 5 },
+  },
+  prerequisites: [
+    [attack, special],
```

Comment:
For future work on prereqs:
Are these arrays always "one of"? Or are there cases of >1?

Reply:
The inner arrays are logical ORs, so you need 1 of those things. The outer array is a logical AND so you need one from each group